pulse_actions/worker.py

```diff
@@ -3,6 +3,8 @@
 import os
 import traceback
 
+from authentication import get_user_and_password
```

Comment:
It *might* need "from __future__ import absolute_import" to make the
absolute import.

On 17 September 2015 at 11:44, Alice <notifications@github.com> wrote:

> In pulse_actions/worker.py
> <https://github.com/adusca/pulse_actions/pull/18#discussion_r39761704>:
>
> > @@ -3,6 +3,8 @@
> >  import os
> >  import traceback
> >
> > +from authentication import get_user_and_password
>
> Can you change this to an absolute import (from
> pulse_actions.authentication import get_user_and_password)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adusca/pulse_actions/pull/18/files#r39761704>.
>



-- 
Zambrano Gasparnian, Armen
Automation & Tools Engineer
http://armenzg.blogspot.ca